test/benchmark/benchmark.go

```diff
@@ -118,6 +126,33 @@ func main() {
 	}
 }
 
+// If there is already a schema present, clear it out
+func clearExistingObjects(c *http.Client, url string) {
+	checkSchemaRequest := createRequest(url+"schema", "GET", nil)
+	checkSchemaResponseCode, body, _, err := performRequest(c, checkSchemaRequest)
+	if err != nil {
+		panic("perform request: %v\n" + err.Error())
```

Comment:
you can also here wrap the error like this:
```
panic(errors.Wrap(err, "perform request"))
```

Reply:
I can change this, but why is this better? :)

Reply:
For this case it most likely won't matter, but in general the explanation would be along those lines:

If you have a specific error that you want to handle differently than other errors you can check for such an error using `errors.Is()`. When you concatenate strings, this information gets lost, but when you use `errors.Wrap()`, `errors.Is()` will match even on errors that have been wrapped multiple times.

In this case the error is passed straight to panic, so it's unlikely it's ever going to be handled, but I agree that `errors.Wrap()` is very good practice to use *in general*.

Reply:
oki, changed it everywhere